frontend/catalyst/third_party/oqc/src/tests/CMakeLists.txt

```diff
@@ -34,7 +34,6 @@ target_link_directories(runner_tests_oqc PRIVATE ${runtime_lib})
 
 target_link_libraries(runner_tests_oqc PRIVATE
     Catch2::Catch2
-    pybind11::embed
```

Comment:
I'm assuming the configuration here will need to be similar to that in the runtime, where for the C++ tests we still use pybind11 in order to embed the interpreter. Since the tests are not shipped in the wheel this doesn't have to conform to the stable ABI though.

Reply:
Done